crates/revm/src/custom_upcode.rs

```diff
@@ -0,0 +1,253 @@
+use revm_interpreter::{
```

Comment:
This file merely serves to illustrate that we can support custom opcodes. It should be removed before merging.

Reply:
We can move it to examples later, fine with leaving it here for now.